changelog/20.0/20.0.0/summary.md

```diff
@@ -325,6 +326,12 @@ VTTablet exposes two new counter stats:
  * `QueryCacheHits`: Query engine query cache hits
  * `QueryCacheMisses`: Query engine query cache misses
 
+### <a id="#vttablet-query-text-characters-processed"/>VTTablet Query Text Characters Processed
+
+VTGate and VTTablet exposes a new counter stat `QueryTextCharactersProcessed` to reflect the number of query text characters processed.
+
+VTGate groups this metric by Operation, Keyspace and DbType. On VTTablet it is grouped by Table, Plan and optionally Workload.
```

Comment:
What is DbType? Looking at how `statsKey` is constructed, this seems to be TabletType. We use TabletType everywhere in the docs, so it would be confusing to add a new term.